CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### Features
 
+* [#2375](https://github.com/ruby-grape/grape/pull/2375): Fix setter methods for `Grape::Router::AttributeTranslator` - [@Jell](https://github.com/Jell)
```

Comment:
It's still a fix even though it never worked. Move it down to fixes?

Reply:
It's also missing a period at the end ;)

Reply:
ah of course 🤦

Reply:
done!